modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/identityServlet/ExtendSessionEndpointAuthCodeGrantTestCase.java

```diff
@@ -271,7 +270,8 @@ public void testSessionExtensionWithValidParameters() throws Exception {
     public void testRefreshGrantIdTokenClaimAvailability() throws Exception {
 
         RefreshTokenGrant refreshGrant = new RefreshTokenGrant(refreshToken);
-        OIDCTokens oidcTokens = makeTokenRequest(refreshGrant, ACCESS_TOKEN_ENDPOINT, OAUTH2_SCOPE_OPENID);
+        OIDCTokens oidcTokens = makeTokenRequest(refreshGrant, ACCESS_TOKEN_ENDPOINT,
```

Comment:
Need to assert that the requested scopes are returned in the token response?
Is this happening now?

Reply:
Currently, this hasn't been explicitly asserted. Will make the change.

Reply:
+1